### PR TITLE
Update bundle structure and dependencies

### PR DESCRIPTION
**Pull request by Jeroeny**
- Rename library to `teamdotblue/psysh-bundle`
- Support `psy/psysh": "^0.11 || ^0.12"`
- Support Symfony 7
- Add PHP-CS Fixer
- Add PHPStan
- Update PHPUnit to `^10`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,49 +1,59 @@
...
-
     "require": {
-        "php": ">=8.0",
-        "psy/psysh": "^0.11",
-        "symfony/error-handler": "^5.4|^6.0",
-        "symfony/expression-language": "^5.4|^6.0",
-        "symfony/framework-bundle": "^5.4|^6.0"
+        "php": ">=8.2",
+        "psy/psysh": "^0.11 || ^0.12",
+        "symfony/error-handler": "^6.4 || ^7.0",
```
Comment (xvilo) [inline composer.json:17]: Symfony 5.4 is still maintained until November 2025 and we might need that support too, would be great if we could keep it in

File: composer.json [hunk 2]
```diff
@@ -1,49 +1,59 @@
...
+        "friendsofphp/php-cs-fixer": "^3.0",
+        "phpstan/phpstan": "^1.0",
+        "phpunit/phpunit": "^10.5",
+        "rector/rector": "^1.0"
     },
-
     "autoload": {
         "psr-4": {
-            "Fidry\\PsyshBundle\\": "src"
+            "TeamBlue\\PsyshBundle\\": "src"
```
Comment (xvilo) [inline composer.json]: General conventions regarding composer packages want us to re-use the github username, so please use `TeamDotBlue\\PsyshBundle` instead?

File: config/services.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        ->alias('psysh.shell', Shell::class)
+
+        ->set(PsyshCommand::class)
+        ->alias('psysh.command.shell_command', PsyshCommand::class)
+
+        ->set(PsyshFacade::class)
+            ->public()
+            ->call('setContainer', [service('service_container')])
+        ->alias('psysh.facade', PsyshFacade::class)
+
```
Comment (xvilo) [inline config/services.php]: Multiple new lines here, I think one can go

File: composer.json (hunk 1 above)
Comment (Jeroeny) [reply composer.json:17]: 4.5.0 can still be installed  for that. For new features I suggest to only support one LTS Symfony version. It's FOSS after all.

**Review by xvilo — APPROVED** [review]

**Pull request merged**